### DISCUSSION TITLE
ChangeAnim3 and Explod ReadPlayerId and ReadPlayerIdAnim

### DISCUSSION THREAD

**Discussion by TRRESTGHGYTCFTGBCXSZD** [ikemen-engine/Ikemen-GO#2169] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

I'm always frustrated on some characters refuse to change stateowner, typically paired with statedef -3 and statedef -2.
Can't change anim of yourself but use opponent's sprites, like n'doul's stage, which the bars are different from typical fighting scene.
Helpers must be in custom stated in order to properly work, and as the reasons above, there are no guarantee it does work.

### Describe the solution you'd like

Being able to change animations of yourself and sprites of opponent, like an opposite version of ChangeAnim2, with explods that use opponent's sprite but it's own animation.

### Additional context

For Example:
```zss
ChangeAnim3{value: 7240, readplayerid: enemy, id} #changes the anim into player's anim 7240, but use enemy's sprites.
#readplayerid is important, because without it, it simply dosen't work.

Explod{anim: 7240, readplayerid: enemy, id, readplayeridanim: id} #puts explod with player's anim 7240, but use enemy's sprites.
```

n'doul's stage:
![0000](https://github.com/user-attachments/assets/907c794b-2499-41cb-8a41-e2368cf6c55e)

### Search terms

ChangeAnim

**Comment by rakieldev** [comment, marked as answer]
We now have ``animPlayerNo`` and ``spritePlayerNo`` parameters implemented for several sctrls (and as triggers), so this is already possible without needing an "changeAnim3"
